### PR TITLE
Update standards.rst

### PR DESCRIPTION
**Pull request by VolCh**
According http://symfony.com/blog/new-in-symfony-3-2-runtime-environment-variables

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
👍

**Review by roman-1983 — APPROVED** [review]
👍

File: contributing/code/standards.rst [hunk 1]
```diff
@@ -218,7 +218,8 @@ Service Naming Conventions
 
 * The DI alias of the bundle is the first group (e.g. ``fos_user``);
 
-* Use lowercase letters for service and parameter names;
+* Use lowercase letters for service and parameter names (except when referring
+  to environment variables with the ``%env(VARIABLE_NAME)%`` syntax);
```
Comment (HeahDude) [inline contributing/code/standards.rst]: Maybe we should note that they should not begin with an underscore as it is to be deprecated.

**Comment by xabbuh** [comment]
Thank you @VolCh.

**Pull request merged**